scylladb/README.md

```diff
@@ -0,0 +1,35 @@
+# Summary
+
+This directory includes a dockerfile to create a patched version of scylladb so
+it will run in chromeos. Without it, scylladb
+[crashes with a SIGFPE error from hwloc](https://github.com/scylladb/scylla/issues/10439).
+This is due to a
+[chromeos bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1304418).
+Fortunately, hwloc has released a
+[patch](https://github.com/open-mpi/hwloc/commit/33b555b5a1c8339daeb4215bad57430b57a6b33f)
+in 2.7.1.
+
+## Rebuild the libhwloc.so file
+
+Go to https://www.open-mpi.org/software/hwloc/v2.7/ and get the latest release.
+Replace `${HWLOC_VERSION}` below with the actual release version
+
+```shell
+export HWLOC_VERSION="2.7.1"
+curl --proto "=https" \
+    --tlsv1.2 \
+    --location \
+    --output /tmp/hwloc-${HWLOC_VERSION}.tar.gz \
+"https://download.open-mpi.org/release/hwloc/v2.7/hwloc-${HWLOC_VERSION}.tar.gz"
+cd /tmp
+tar -vxzf hwloc-${HWLOC_VERSION}.tar.gz
+cd hwloc-${HWLOC_VERSION}
+./configure
+make
+make install
```

Comment:
I was going to suggest creating an actual script for this, so it would be executable. Another possibility would be to fold this into the Dockerfile itself (e.g., in a different target). I'm not sure how long it takes to build, but if it's not too long, it might be good to remove some additional moving parts (including the need to store anything in Cloudsmith, not that that's a horrible thing to do, of course).

Reply:
Moving into the Dockerfile as a first stage makes sense to me

Reply:
I've moved it into the Dockerfile